spec/integration/streams/single/grow_dimension_spec.rb

```diff
@@ -115,4 +115,41 @@
       expect(new_edition).to have_attributes(warehouse_item_id: warehouse_item_id, base_path: new_base_path)
     end
   end
+
+  context 'when handling unpublishing related messages' do
+    it 'it does not grow dimensions_editions unnecessarily' do
+      edition = create :edition
+
+      first_gone_message = build(
```

Comment:
What about extracting this `gone` message into a factory so we can reuse it?

Reply:
Sure, I've made an attempt at that now :)